formats/polus-tiledtiff-converter-plugin/README.md

```diff
@@ -50,12 +53,45 @@ This plugin takes one input argument and one output argument:
 | `input`       | Input image collection        | Input  | Path    | true     |
 | `output`      | Output image collection       | Output | Path    | true     |
 
-## Run the plugin
+## Example Code
 
-### Run the Docker Container
-
-```bash
-docker run -v /path/to/data:/data labshare/polus-tiledtiff-converter-plugin:1.1.0 \
-  --input /data/input \
-  --output /data/output
+```Linux
+mkdir examples
+cd examples
+mkdir output
+wget -P images/ https://data.broadinstitute.org/bbbc/BBBC033/BBBC033_v1_dataset.zip
+unzip images/BBBC033_v1_dataset.zip
+basedir=$(basename ${PWD})
+docker run -v ${PWD}:/$basedir labshare/polus-tiledtiff-converter-plugin:1.1.0 \
+--input /$basedir/"images/" \
+--output /$basedir/"output/"
 ```
+
+Navigate to the `examples/output/` directory to visualize the tiles using a 
+tool such as bfio.
+
+## Viewing the results using Python
+
```

Comment:
It would be a good idea to include a line here about installing dependencies. For example, it is necessary to install `bfio` for your code below.